assets/index.html

```diff
@@ -40,7 +40,7 @@ <h1>Blog.</h1>
             <div id="blogs"></div>
         </div>
         <div id="footer">
-            <a href="https://github.com/imfunniee" target="_blank">made on earth by a human</a>
```

Comment:
Why is the credit to the original author removed?

Reply:
ikr